src/context/CapturesContext.js

```diff
@@ -150,15 +166,26 @@ export function CapturesProvider(props) {
         morphology: true,
         captureApprovalTag: true,
         rejectionReason: true,
+        uuid: true,
+        imageUrl: true,
+        lat: true,
+        lon: true,
+        timeUpdated: true,
+        note: true,
       },
     };
 
     const paramString = `filter=${JSON.stringify(lbFilter)}`;
-    const response = await queryCapturesApi({ paramString });
-    setCaptures(response.data);
+    log.debug('load page with params:', paramString);
+    const { data } = await queryCapturesApi({ paramString }).catch((err) =>
+      console.error(`ERROR: Failed to get captures: ${err}`)
+    );
+    log.debug('loaded captures:', data.length);
+    setCaptures(data);
+    setIsLoading(false);
```

Comment:
Can this be chained together? To something like:
```javascript
     await queryCapturesApi({ paramString })
        .then((data) => { 
             log.debug('loaded captures:', data.length);
             setCaptures(data);
             setIsLoading(false);
        })
        .catch((err) => console.error(`ERROR: Failed to get captures: ${err}`)
    );

```

Reply:
Ah, nice -- thanks!